spec/restaurant_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'item'
+require 'customer'
+
+describe 'restaurant' do
+
+  subject {Restaurant.new}
+
+  it 'creates a restaurant' do
+    expect(subject).to be_kind_of Restaurant
+  end
+
+    it 'creates a menu' do
+    expect(subject.menu).to be_kind_of Hash
+  end
+
+    it 'menu contains dishes when created' do
+    expect(subject.menu[:nr1]).to be_kind_of Item
+  end
```

Comment:
Lint/BlockAlignment: `end` at 18, 2 is not aligned with `it 'menu contains dishes when created' do` at 16, 4